src/ui/atoms/Image.tsx

```diff
@@ -1,21 +1,28 @@
+import clsx from "clsx";
 import NextImage from "next/image";
 
 type Props = {
 	src: string;
 	alt: string;
 	width: number;
 	height: number;
+	square?: boolean;
+	fit?: "object-contain" | "object-cover";
```

Comment:
Is this used anywhere with `object-cover`?

Reply:
Right now nowhere. I just wanted to give a posibility to choose between this two options. Sometimes, it looks good when images are displayed in a container with `object-fit: cover;` but on the product page, we want to see the entire product image, so the ideal choice is to display the image with `object-fit: contain;` - you can look at the examples in PR description. Maybe I add this props exaggeratedly and I should leave hardcoded only one option, I think `object-fit: contain;` should be the best. What do you think?

Reply:
This component has been now modified and accepts any props

Reply:
Ok, after consideration, I've opted to stick with `object-fit: contain;` because, from the user's perspective, it's crucial to see the entire product, which may not always be possible when the image has the `object-fit: cover;` property. 
If there is a need to present the image in a different manner, then it will be an appropriate moment to introduce additional options to the component responsible for displaying the image.

Reply:
@mmiszy sorry, I didn't see your comment because I had the stale page open.
Newly created component that receive any props have a default set of classes. So, should I extend them or override these classes in the places where we need to resolve this issue? In your opinion, which approach will be better? Now I simply modified default set of classes. Let me know if shoul I change it.